katas/langtonant/alfredo/module1.test.js

```diff
@@ -0,0 +1,32 @@
+import { class1 } from './module1.js';
```

Comment:
These tests are good. Try to add the ones for the main functionality as well: given a number of steps, what would be the final status of the ant